pot-luck-fe/src/App.js

```diff
@@ -22,28 +23,23 @@ const NavBar = styled.header`
 
 function App() {
   return (
-
-
-
     <div className="App">
       <NavBar>
-        <h1>Placeholder login bar</h1>
+        <Link to="/potlucks">Potlucks Page</Link>
       </NavBar>
 
-   
-
-
       <Switch>
-          <Route exact path="/">
-          <UserPotluckPage /> 
-          </Route>
-          <Route  path="/signup" >
-            <SignUp />
-          </Route>
-          <Route path="/login" >
-            <Login />
-          </Route>
-        </Switch>
+        <Route exact path="/">
+          <Login />
+        </Route>
+        <Route path="/signup">
+          <SignUp />
+        </Route>
+        <Route path="/login">
```

Comment:
Was it intentional to have 2 routes for the login page?

Reply:
It is for the moment. We will update the default route later